app/scripts/controllers/gene.js

```diff
@@ -471,14 +469,7 @@ angular.module('oncokbApp')
                 return '';
             };
             function isChangedSection(uuids) {
-                var result = false;
-                _.some(uuids, function (uuid) {
-                    if (uuid && $scope.geneMeta.review[uuid]) {
-                        result = true;
-                        return true;
-                    }
-                });
-                return result;
+                return _.intersection(uuids, _.keys($scope.geneMeta.review)).length > 0;
```

Comment:
I would not use intersection over some. It has worse performance.

Reply:
I'll roll it back.